_examples/beh-2015-sequencing.md

```diff
@@ -6,64 +6,64 @@ permalink: /examples/beh-2015-sequencing/
 page_authors:
   - Caroline Rosello
   - Joseph Guillaume
-page_reviewers:
-excerpt: The analysis identifies optimal sequences of additional water supply options for a set of scenarios with different climate change, population growth, and discount rates. Sensitivity analysis shows how performance of each plan varies across scenarios, and which variables have greater influence, to help inform selection of an optimal sequence plan.
+page_reviewers: Leila Noble
+excerpt: The analysis identifies optimal sequences for adding to an urban water supply for a set of scenarios with different values for climate change, population and discount rates. Sensitivity analysis shows how the performance of each sequence for multiple criteria varies across the scenarios, and which uncertain variables have greater influence, to help inform decision-makers. 
 approach: >
 <p> 
-The approach consists of the evaluation of the robustness of water supply management plans by considering the sequencing/schedulling of water supply enhancement options (i.e., additional options to support current water supplies) over a planning horizon to cope optimally with future conditions.
```

Comment:
I think robustness is important here. Is there a specific reason it was removed?